Sources/SwifterSwift/SwiftStdlib/BinaryIntegerPointExtensions.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  BinaryIntegerPointExtensions.swift
+//  SwifterSwift
+//
+//  Created by calm on 2019/6/3.
+//  Copyright © 2019 SwifterSwift
+//
+
+#if canImport(Foundation)
+import Foundation
+
+// MARK: - Properties
+public extension BinaryInteger {
+    
+    #if canImport(Foundation)
```

Comment:
Since we are not relying on any Foundation API, I think all of those #if canImport(Foundation) aren't necessary.

Reply:
Yes, so I resubmitted one and removed those `#if canImport(Foundation)`😊